submissions/Levi123/tiny-JS/index.js

```diff
@@ -0,0 +1,106 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: https://github.com/Levi123/a-tiny-JS-world
+   Web app: _put project's github pages URL here_
+   */
+
+// ======== OBJECTS DEFINITIONS ========
+// Define your objects here
+const dog = {
+   species: 'dog',
+   name: 'Rich',
+   gender: 'male',
+   legs: 4,
+   hands: 0,
+   saying: 'Hav-hav!',
+   friends: ['Bonya'],
+}
+
+const cat = {
+   species: 'cat',
+   name: 'Bonya',
+   gender: 'female',
+   legs: 4,
+   hands: 0,
+   saying: 'Mr-mr-mr',
+   friends: ['Rich', 'Catwoman'],
+}
+
+const female = {
+   species: 'human',
+   name: 'Olya',
+   gender: 'female',
+   legs: 4,
+   hands: 4,
+   saying: "Glory to Ukraine",
+   friends: ['Yura', 'Rich'],
+}
+
+const male = {
+   species: 'human',
+   name: 'Yura',
+   gender: 'male',
+   legs: 4,
+   hands: 4,
+   saying: "Glory to Ukraine",
+   friends: ['Olya', 'Bonya'],
+}
+
+const catWoman = {
+   species: 'human',
+   name: 'Catwoman',
+   gender: 'female',
+   legs: 4,
+   hands: 4,
+   saying: cat.saying,
+   friends: ['Bonya']
+}
+// ======== OUTPUT ========
+/* Use print(message) for output.
+   Default tag for message is <pre>. Use print(message,'div') to change containing element tag.
+
+   Message can contain HTML markup. You may also tweak index.html and/or styles.css.
+   However, please, REFRAIN from improving visuals at least until your code is reviewed
+   so code reviewers might focus on a single file that is index.js.
+   */
+
+const prop = ['species', 'name', 'gender', 'legs', 'hands', 'saying', 'friends']
+const allCharactersArray = [catWoman, male, female, cat, dog]
+
+// function printObjectNew(personToPrint){
+//    const propToPrint = [];
+//    prop.forEach((propOfObj) => {
+//       propToPrint.push(personToPrint[propOfObj]);
+//    })
+//    let newStr = propToPrint.join('; ');
+//    print(newStr);
+// }
+
+function arrayToString(propToPrint){
+   let newStr = propToPrint.join('; ');
+   console.log(propToPrint);
+   print(newStr);
+}
+
+function arrayOfPropToPrint(personToPrint){
+   const propToPrint = [];
+   prop.forEach((propOfObj) => {
+      propToPrint.push(personToPrint[propOfObj]);
+   })
+   arrayToString(propToPrint);
+}
+
+function printAllObjects(arrayAllObjects){
+   arrayAllObjects.forEach((characters) => {
+      arrayOfPropToPrint(characters);
+   })
+}
+
+printAllObjects(allCharactersArray);
+
+// printObjectNew(catWoman);
+// printObjectNew(male);
+// printObjectNew(female);
+// printObjectNew(cat);
+// printObjectNew(dog);
```

Comment:
![image](https://user-images.githubusercontent.com/8686631/186651370-c829b2f9-668a-4f22-9e1e-dc9c382c061c.png)

Make your code editor to ensure you always have an empty line at the end of file. There must be a setting.  Check [this](https://github.com/OleksiyRudenko/dev-env-git#ide-minimal-tweak)

Reply:
Thank you! Done